### PR TITLE
Added phpunit problem matcher

### PR DESCRIPTION
**Pull request by staabm**
Similar to https://github.com/rectorphp/rector/pull/2756 this will enable reporting of phpunit test failures into the files-changed PR tab

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Now there are false positives. Error reported for skipped tests

**Comment by staabm** [comment]
Reported the issue upstream in https://github.com/shivammathur/setup-php/issues/163

Not sure it is fixable thought

**Comment by staabm** [comment]
After checking the phpunit sources, I wonder why we see the infos printed regarding skipped tests.
https://github.com/sebastianbergmann/phpunit/blob/b6fcd8290a37d301a4a87abaf2593d5f1ac5bd45/src/TextUI/DefaultResultPrinter.php#L168

These should only be printed in verbose-mode

**Comment by staabm** [comment]
Guess I found it

File: phpunit.xml [hunk 1]
```diff
@@ -3,7 +3,6 @@
     xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
     bootstrap="tests/bootstrap.php"
     colors="true"
-    verbose="true"
```
Comment (staabm) [inline phpunit.xml]: In case you want to stick with verbose mode: the upstream project was fixed and the next version of the gh-action should work then

**Comment by staabm** [comment]
We got an upstream release and things are looking better now

**Comment by staabm** [comment]
Any feedback?

File: .github/workflows/tests.yaml [hunk 2]
```diff
@@ -20,6 +20,10 @@ jobs:
                 with:
                     php-version: 7.2
```
Comment (TomasVotruba) [inline .github/workflows/tests.yaml]: I just notice, here should be `${{ matrix.php }}`

File: .github/workflows/tests.yaml [hunk 3]
```diff
@@ -20,6 +20,10 @@ jobs:
...
                     coverage: none # disable xdebug, pcov
+            # report phpunit errors into files-changed PR tab
```
Comment (TomasVotruba) [inline .github/workflows/tests.yaml]: Link to some knowledge base would help

**Comment by staabm** [comment]
Here you are!

**Comment by TomasVotruba** [comment]
Looks good!
Let's wait for CI and merge then

Thank you for your work :+1:

**Comment by TomasVotruba** [comment]
I'll look how this can be added for Rector

**Pull request merged**